### PR TITLE
Cleanup/remove django pipeline in favour of plain css files

### PR DESCRIPTION
**Pull request by rasca**
We had many headaches with this and weren't exploiting it's usefulness, we decided to remove it to have less magic.

### CODE REVIEW

File: tickets/static/css/global.css [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+  .main-navigation-wrap {
+    top: -100px;
+  }
+}
+
+.emoji {
+  font-size: 2rem;
+}
+
+.list-group .list-group-item {
```
Comment (vfonic) [inline tickets/static/css/global.css:233]: No hacía falta hacer eso, pero bueno:
https://caniuse.com/?search=css%20nesting

File: tickets/static/css/global.css [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+.list-group .list-group-item p {
+  margin: 0;
+}
+
+.list-group .list-group-item .emoji {
+  margin-top: -1.5rem;
+  margin-right: -1rem;
+}
+
+.list-group     .card {
```
Comment (vfonic) [inline tickets/static/css/global.css:247]: Podés auto-formatear eso.

**Review by vfonic — APPROVED** [review]
Gracias!!! 👍 🤘

File: tickets/static/css/global.css (hunk 1 above)
Comment (rasca) [reply tickets/static/css/global.css:233]: Oh! Okey! Lo puedo volver para atrás! gracias

File: tickets/static/css/global.css (hunk 2 above)
Comment (rasca) [reply tickets/static/css/global.css:247]: Perfect

**Pull request merged**